src/components/new-request/search.vue

```diff
@@ -182,10 +182,8 @@
     <v-row no-gutters>
       <v-col cols="12">
         <template>
-          <!-- Search for business identifier or name -->
-          <BusinessLookup
-            @business="businessName = $event.name; businessIdentifier = $event.identifier"
```

Comment:
When you removed the event handling (assigning businessName and businessIdentifier), I thought you were also going to delete the display block below (and local variables) ...?

Reply:
Yes

Reply:
I removed that. But BusinessLookup is hidden when we select one of the results.
If you are okay, I am going to remove the condition block here for now.

https://github.com/bcgov/namerequest/blob/1df5453e6dc37c47293dc0c6e20913d0bcd7bc27/src/components/new-request/business-lookup.vue#L3

What do you think?

Reply:
I think you should leave the Business Lookup component as functional as possible for future work, so don't disable it. Try to make it demo-able without impacting the flows that we are working on.

Reply:
okay